Cython/Build/Tests/TestIpythonMagic.py

```diff
@@ -14,6 +14,7 @@
 
 try:
     import IPython.testing.globalipapp
+    from IPython.utils.io import capture_output
```

Comment:
Are you sure that this does not interfere with the I/O capturing on our side?

Reply:
It looks as if it only replaces sys.stdout/stderr so it should not interfere with C-level redirection.